### ISSUE TITLE
Toolbar icons not showing in windows

### ISSUE THREAD

**Issue by JonSilas** [cbrnr/mnelab#384]
Hello, 

I just picked up using NME lab - I think it may be a really useful teaching tool for our students getting started with EEG lab. 

On my Windows 11 machine, I'm unable to see any of the icons in the toolbar (see image) - is this unique to me? 

![image](https://github.com/cbrnr/mnelab/assets/120580538/ac54c31b-6a54-45a6-9e78-ad25b77d6460)

Many thanks in advance! 
J

**Comment by cbrnr** [comment]
This keeps popping up, which version of MNELAB are you using and how did you install it?

**Comment by JonSilas** [comment]
Thanks. I'm using MNELAB 0.8.6 and installed using `pip install mnelab`.

**Comment by JonSilas** [comment]
Actually, I've been starting with `python - mnelab`. On its own, `mnelab`  returns an error: 
```
mnelab : The term 'mnelab' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ mnelab
+ ~~~~~~
    + CategoryInfo          : ObjectNotFound: (mnelab:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
``` 
Quite likely something I'm doing wrong - new MNE and MNElab! 
Thanks.

**Comment by cbrnr** [comment]
Toolbar icons are visible on my Windows 11 machine (both `mnelab` and `python -m mnelab` work), so I assume the problem has something to do with your Python installation.

Which Python version did you install? How did you install Python? It looks like you used the official installer from python.org and installed it for all users. This is unfortunately a bit broken, because you need to manually add the following folder to your path:

```
%USERPROFILE%\AppData\Roaming\Python\Python311\Scripts
```

After that, you should be able to start MNELAB with `mnelab`.

Can you share a screenshot of the About MNELAB dialog? This dialog lists versions for required and optional dependencies.

**Comment by JonSilas** [comment]
Gah - sorry, I was running an old version of python (3.8) and I'm now up-to-date and seeing everything as expected. 

I installed the official Python 3.11 using the Windows installer, reinstalled mnelab using `pip install mnelab` and I manually added:

```
%USERPROFILE%\AppData\Roaming\Python\Python311\Scripts
```
but `mnelab` produces the same error. I still need to use `python -m mnelab`.

Thanks for your support!

Screenshot of about dialogue: 
![image](https://github.com/cbrnr/mnelab/assets/120580538/a0d8dc8b-0512-43b4-82ac-b2a3bc18456f)
